go.mod

```diff
@@ -21,6 +20,7 @@ require (
 	k8s.io/pod-security-admission v0.0.0
 	k8s.io/utils v0.0.0-20230209194617-a36077c30491
 	sigs.k8s.io/sig-storage-lib-external-provisioner/v6 v6.3.0
+	sigs.k8s.io/yaml v1.3.0
```

Comment:
Does vendor need to be updated with the new package?

Reply:
It was actually an indirect dependency before this pull request was made. Therefore, it has already been included in the vendor directory. https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/56a9cf7443ccc10359569ca229a10035205d4e26/vendor/sigs.k8s.io/yaml

https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/56a9cf7443ccc10359569ca229a10035205d4e26/go.mod#L133